packages/twenty-front/src/pages/settings/data-model/SettingsObjectEdit.tsx

```diff
@@ -16,6 +16,7 @@ import { RecordFieldValueSelectorContextProvider } from '@/object-record/record-
 import { SaveAndCancelButtons } from '@/settings/components/SaveAndCancelButtons/SaveAndCancelButtons';
 import { SettingsPageContainer } from '@/settings/components/SettingsPageContainer';
 import {
+  IS_LABEL_SYNCED_WITH_NAME_LABEL,
```

Comment:
not 100% sure this const should be specifically owned by this component? Should we move to a constants folder instead?